docs/en/stock-management/stock.settings.md

```diff
@@ -0,0 +1,66 @@
+&raquo; [Home](../index.md) / [Inventory Management](./index.md) / Stock Settings
+
+# Stock Settings
+
+There a number of settings that affect how the BHIMA software deals with stock
+management.  To check and update the settings, one goes to the "Stock
+Settings" page (under the "Stock" menu):
+
+![Stock Settings Menu](./images/stock-settings-menu.png)
+
+If you do not see the "Stock Settings" menu item, consult your system
+adminstrator.  You need to be given permission to access that page.
+
+- To add permission for a user to access the "Stock Settings' page: Under
+  `Administration > Role Management`, click on the action menu on the right
+  end of the desired user.  In the action menu, click on `Permission`.  Scroll
+  down to "Stock Settings" (at the end of the Stock items), then click the check
+  box and the [Save] button.  The Stock Settings page should now be accessible
+  for the user.
+
+The Stock Settings Page allows access to the following items:
+
+- **Number of months for calculating the average monthly consumption** (integer)
+
+- **Default minimum number of months of security stock for depots** (integer)
+
+  - **Enable automatic confirmation of purchase orders** (yes/no).  
```

Comment:
I really like the "yes/no" options.  It might be nice to do what unixes do on the command line and capitalize the default option.  (yes/NO).   You'll find the default option in the schema.sql.

Reply:
Done.